### DISCUSSION TITLE
Gehelper wont run

### DISCUSSION THREAD

**Discussion by Noob69x** [seerge/g-helper#1766] [Q&A]
Hi sorry to bother but im new to all this pc and everything. I downloaded the latest version of gehelper extracted it to a folder and while the app is visible in the folder no matter how many times i click it it doesnt run. I also tried running in administrator, and i have both the net runtime and the asus v3 display installed

**Comment by Kikimahe** [comment]
Hi, 

Try temporarily disabling 'Microsoft defender Smartscreen'.

![Capture d'écran 2023-12-20 202244](https://github.com/seerge/g-helper/assets/142881210/0262a80f-0845-40aa-9f65-f7b678a83a31)
